osu.Game/Storyboards/Drawables/DrawableStoryboardAnimation.cs

```diff
@@ -128,7 +128,7 @@ protected override void LoadComplete()
             //
             // In the case of storyboard animations, we want to synchronise with game time perfectly
             // so let's get a correct time based on gameplay clock and earliest transform.
-            PlaybackPosition = gameplayClock.CurrentTime - Animation.EarliestTransformTime;
+            PlaybackPosition = (beatSyncProvider?.Clock?.CurrentTime ?? Clock.CurrentTime) - Animation.EarliestTransformTime;
```

Comment:
Is there any way for a beat sync provider to be `null` if it's inherited by `OsuGameBase`? I've noticed in all other usages that it's DI'd as nullable while here a null conditional is added while the dependency is resolved as non-nullable..?

Reply:
I was just being extra safe. It should not be null, so it's probably fine to remove that check (although maybe no harm?).

The `Clock` null check still needs to be there so it won't change too much.